user-carbon/src/BCT.ts

```diff
@@ -0,0 +1,13 @@
+import { ZERO_ADDRESS } from "../../lib/utils/Constants";
```

Comment:
Since `transfer` method is being handled from each ERC20 token, maybe we could do something similar with bonds - having one `TransferHandler.ts` file where we could fetch Token class via factory where (`event.address` would be a parameter)? 
This way there would be no need to create a new handler file for new Carbon Token - we would have to create a new `Token` implementation and modify `subgraph.yaml` and that should be it